### PR TITLE
Fixed tooltip in security

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description
Fixed the tooltips of the tables in the security section, also removed requests that were made when opening and closing flyouts without making any changes.
 
### Issues Resolved
- #5267 

### Evidence

image

image

### Test

[tests.md](https://github.com/wazuh/wazuh-kibana-app/files/11938277/tests.md)

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: plugins/main/public/components/security/policies/create-policy.tsx [hunk 1]
```diff
@@ -171,7 +171,7 @@ export const CreatePolicyFlyout = ({ closeFlyout }) => {
           effect: effectValue,
         },
       });
-      const resultData = (result.data || {}).data;
+      const resultData = result?.data?.data;
       if (resultData.failed_items && resultData.failed_items.length) {
```
Comment (jbiset) [inline plugins/main/public/components/security/policies/create-policy.tsx:175]: If resultData is undefined, there may be an error trying to read failed_items from undefined. Right?

**Review by jbiset — COMMENTED** [review]
# Wazuh dashboard

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Delete a user that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to edit a user and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to create a user and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Delete a role that is above a reserved role in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to edit a role and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to create a role and closing it without creating any role should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Delete a policy that is above a reserved policy in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to edit a policy and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to create a policy and closing it without creating any policy should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Delete a roles mapping that is above a reserved roles mapping in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to edit a roles mapping and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to create a roles mapping and closing it without creating any role mapping should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |

**Details**
<details>
<summary>:green_circle: Delete a user that is above a reserved user in the table, and see that the tooltip does not appear.</summary>

Chrome - :green_circle:
test1_Chrome

Firefox - :green_circle:
test1_Firefox

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Opening the flyout to edit a user and closing it without making any changes should not trigger any requests.</summary>

Chrome - :green_circle:
test2_Chrome

Firefox - :green_circle:
test2_Firefox

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Opening the flyout to create a user and closing it without creating any user should not trigger any requests.</summary>

Chrome - :green_circle:
test3_chrome

Firefox - :green_circle:
test3_Firefox

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Delete a role that is above a reserved user in the table, and see that the tooltip does not appear.</summary>

Chrome - :green_circle:
test4_Chrome

Firefox - :green_circle:
test4_Firefox

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Opening the flyout to edit a role and closing it without making any changes should not trigger any requests.</summary>

Chrome - :green_circle:
test5_Chrome

Firefox - :green_circle:
test5_Firefox

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Opening the flyout to create a role and closing it without creating any role should not trigger any requests.</summary>

Chrome - :green_circle:
test6_Chrome

Firefox - :green_circle:
test6_Firefox

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Delete a policy that is above a reserved policy in the table, and see that the tooltip does not appear.</summary>

Chrome - :green_circle:
test7_Chrome

Firefox - :green_circle:
test7_Firefox

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Opening the flyout to edit a policy and closing it without making any changes should not trigger any requests.</summary>

Chrome - :green_circle:
test8_Chrome

Firefox - :green_circle:
test8_Firefox

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Opening the flyout to create a policy and closing it without creating any policy should not trigger any requests.</summary>

Chrome - :green_circle:
test9_Chrome

Firefox - :green_circle:
test9_Firefox

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Delete a roles mapping that is above a reserved role mapping in the table, and see that the tooltip does not appear.</summary>

Chrome - :green_circle:
test10_Chrome

Firefox - :green_circle:
test10_Firefox

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Opening the flyout to edit a roles mapping and closing it without making any changes should not trigger any requests.</summary>

Chrome - :green_circle:
test11_Chrome

Firefox - :green_circle:
test11_Firefox

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Opening the flyout to create a roles mapping and closing it without creating any role mapping should not trigger any requests.</summary>

Chrome - :green_circle:
test12_Chrome

Firefox - :green_circle:
test12_Firefox

Safari - :black_circle:

</details>

**Review by chantal-kelm — APPROVED** [review]
| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Delete a user that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to edit a user and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to create a user and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Delete a role that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to edit a role and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to role a user and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Delete a policy that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to edit a policy and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to create a policy and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Delete a roles mapping that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to edit a roles mapping and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to create a roles mapping and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |

<summary>Delete a user that is above a reserved user in the table, and see that the tooltip does not appear.</summary>
<details>
Captura de pantalla 2023-07-06 a la(s) 16 32 51
Captura de pantalla 2023-07-07 a la(s) 08 20 19
</details>

<summary>Opening the flyout to edit a user and closing it without making any changes should not trigger any requests.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/a3cdeb94-b431-4a49-b1a0-a36099708b93


</details>

<summary>Opening the flyout to create a user and closing it without creating any user should not trigger any requests. </summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/d36bfea1-6989-46c0-952f-9e42af596427




</details>

<summary>Delete a role that is above a reserved user in the table, and see that the tooltip does not appear.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/e98b1faf-7b58-42fa-8901-64353a3dfd37


</details>

<summary>Opening the flyout to edit a role and closing it without making any changes should not trigger any requests.</summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/73fff095-7be7-4774-9180-d08ab2f2b660




</details>

<summary>Opening the flyout to role a user and closing it without creating any user should not trigger any requests. </summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/6f9a0d54-d022-4fd2-9546-19ea4298498f


</details>

<summary>Delete a policy that is above a reserved user in the table, and see that the tooltip does not appear.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/ae8ccfd8-308b-42ad-9487-5e32d1c094e9


</details>

<summary>Opening the flyout to edit a policy and closing it without making any changes should not trigger any requests.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/b1b5b33a-916f-4844-9239-e0b39e39631b


</details>

<summary>Opening the flyout to create a policy and closing it without creating any user should not trigger any requests.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/3cba3ad6-b4e3-4107-a451-cb851375463a


</details>

<summary>Delete a roles mapping that is above a reserved user in the table, and see that the tooltip does not appear.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/482ced6d-36a7-43e7-93fd-61502deafae1


</details>

<summary>Opening the flyout to edit a roles mapping and closing it without making any changes should not trigger any requests.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/f5e23b82-9656-4c32-a3cd-d60b2e9597d5


</details>

<summary>Opening the flyout to create a roles mapping and closing it without creating any user should not trigger any requests.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/f3b74847-44d2-40fd-8587-36aada7c9675


</details>

**Review by jbiset — DISMISSED** [review]
LGTM
CR: :white_check_mark:  (detail commented)
Test: :heavy_check_mark:

File: plugins/main/public/components/security/policies/create-policy.tsx (hunk 1 above)
Comment (yenienserrano) [reply plugins/main/public/components/security/policies/create-policy.tsx]: Done

File: plugins/main/public/components/security/policies/policies-table.tsx [hunk 2]
```diff
@@ -36,10 +42,13 @@ export const PoliciesTable = ({
             },
           },
         );
-        const data = (response.data || {}).data;
+        const data = response?.data?.data;
```
Comment (asteriscos) [inline plugins/main/public/components/security/policies/policies-table.tsx:45]: Be mindful that data can be `undefined`.

**Review by asteriscos — CHANGES_REQUESTED** [review]
CR: :x: 
There are a few potential falsy variables that may cause an issue.

File: plugins/main/public/components/security/policies/policies-table.tsx (hunk 2 above)
Comment (yenienserrano) [reply plugins/main/public/components/security/policies/policies-table.tsx:45]: done

File: plugins/main/public/components/security/roles/edit-role.tsx [hunk 3]
```diff
@@ -120,7 +119,7 @@ export const EditRole = ({ role, closeFlyout }) => {
         },
       );
 
-      const policiesData = (policyResult.data || {}).data;
+      const policiesData = policyResult?.data?.data;
       if (policiesData.failed_items && policiesData.failed_items.length) {
```
[the remark this answers is not in the document]
Comment (yenienserrano) [reply plugins/main/public/components/security/roles/edit-role.tsx:123]: done

File: plugins/main/public/components/security/roles/roles-table.tsx [hunk 4]
```diff
@@ -34,15 +40,18 @@ export const RolesTable = ({
...
       try {
-        const response = await WzRequest.apiReq('DELETE', `/security/roles/`, {
+        const response = await WzRequest.apiReq('DELETE', '/security/roles/', {
           params: {
             role_ids: item.id,
           },
         });
-        const data = (response.data || {}).data;
+        const data = response?.data?.data;
         if (data.failed_items && data.failed_items.length) {
```
[the remark this answers is not in the document]
Comment (yenienserrano) [reply plugins/main/public/components/security/roles/roles-table.tsx:49]: done

**Review by asteriscos — DISMISSED** [review]

**Review by jbiset — DISMISSED** [review]
LGTM
CR: :heavy_check_mark:

**Review by jbiset — APPROVED** [review]
CR: :heavy_check_mark:  

## UI :heavy_check_mark: 

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Delete a user that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to edit a user and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to create a user and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Delete a role that is above a reserved role in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to edit a role and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to create a role and closing it without creating any role should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Delete a policy that is above a reserved policy in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to edit a policy and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to create a policy and closing it without creating any policy should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Delete a roles mapping that is above a reserved roles mapping in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to edit a roles mapping and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |
| Opening the flyout to create a roles mapping and closing it without creating any role mapping should not trigger any requests. | :green_circle: | :green_circle: | :black_circle: |

**Details**
<details>
<summary>:green_circle: Delete a user that is above a reserved user in the table, and see that the tooltip does not appear.</summary>

test1_A

test1_B

</details>

<details>
<summary>:green_circle: Opening the flyout to edit a user and closing it without making any changes should not trigger any requests.</summary>

test2

</details>

<details>
<summary>:green_circle: Opening the flyout to create a user and closing it without creating any user should not trigger any requests.</summary>

test3

</details>

<details>
<summary>:green_circle: Delete a role that is above a reserved user in the table, and see that the tooltip does not appear.</summary>

test5_A

test5_B

</details>

<details>
<summary>:green_circle: Opening the flyout to edit a role and closing it without making any changes should not trigger any requests.</summary>

test6

</details>

<details>
<summary>:green_circle: Opening the flyout to create a role and closing it without creating any role should not trigger any requests.</summary>

test7

</details>

<details>
<summary>:green_circle: Delete a policy that is above a reserved policy in the table, and see that the tooltip does not appear.</summary>

test8_A

test8_B

</details>

<details>
<summary>:green_circle: Opening the flyout to edit a policy and closing it without making any changes should not trigger any requests.</summary>

test9

</details>

<details>
<summary>:green_circle: Opening the flyout to create a policy and closing it without creating any policy should not trigger any requests.</summary>

test10

</details>

<details>
<summary>:green_circle: Delete a roles mapping that is above a reserved role mapping in the table, and see that the tooltip does not appear.</summary>

test11_A

test11_B

</details>

<details>
<summary>:green_circle: Opening the flyout to edit a roles mapping and closing it without making any changes should not trigger any requests.</summary>

test12

</details>

<details>
<summary>:green_circle: Opening the flyout to create a roles mapping and closing it without creating any role mapping should not trigger any requests.</summary>

test13

</details>

**Comment by chantal-kelm** [comment]
| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Delete a user that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to edit a user and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to create a user and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Delete a role that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to edit a role and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to role a user and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Delete a policy that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to edit a policy and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to create a policy and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Delete a roles mapping that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to edit a roles mapping and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to create a roles mapping and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |

<summary>Delete a user that is above a reserved user in the table, and see that the tooltip does not appear.</summary>
<details>
Captura de pantalla 2023-07-06 a la(s) 16 32 51
Captura de pantalla 2023-07-07 a la(s) 08 20 19
</details>

<summary>Opening the flyout to edit a user and closing it without making any changes should not trigger any requests.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/de0f25c8-e9ac-4853-9e43-7795dc96dc95


</details>

<summary>Opening the flyout to create a user and closing it without creating any user should not trigger any requests. </summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/7e9400b1-c4d9-4f1c-a3d4-8c153d952095


</details>

<summary>Delete a role that is above a reserved user in the table, and see that the tooltip does not appear.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/e98b1faf-7b58-42fa-8901-64353a3dfd37


</details>

<summary>Opening the flyout to edit a role and closing it without making any changes should not trigger any requests.</summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/73fff095-7be7-4774-9180-d08ab2f2b660




</details>

<summary>Opening the flyout to role a user and closing it without creating any user should not trigger any requests. </summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/901037c9-eb06-44c2-acf3-9cc797dca679




</details>

<summary>Delete a policy that is above a reserved user in the table, and see that the tooltip does not appear.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/ae8ccfd8-308b-42ad-9487-5e32d1c094e9


</details>

<summary>Opening the flyout to edit a policy and closing it without making any changes should not trigger any requests.</summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/37eb29f5-5ae6-42f3-9ebb-7a6583d2cd56




</details>

<summary>Opening the flyout to create a policy and closing it without creating any user should not trigger any requests.</summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/31c8142b-2528-4b58-8298-97a2390146bc




</details>

<summary>Delete a roles mapping that is above a reserved user in the table, and see that the tooltip does not appear.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/482ced6d-36a7-43e7-93fd-61502deafae1


</details>

<summary>Opening the flyout to edit a roles mapping and closing it without making any changes should not trigger any requests.</summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/3f7430ac-36d3-4907-a084-c248b8665466




</details>

<summary>Opening the flyout to create a roles mapping and closing it without creating any user should not trigger any requests.</summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/a60e0b79-29a1-4ef5-8654-382ef63b5fc6




</details>

**Review by chantal-kelm — APPROVED** [review]
| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Delete a user that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to edit a user and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to create a user and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Delete a role that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to edit a role and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to role a user and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Delete a policy that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to edit a policy and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to create a policy and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Delete a roles mapping that is above a reserved user in the table, and see that the tooltip does not appear. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to edit a roles mapping and closing it without making any changes should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |
| Opening the flyout to create a roles mapping and closing it without creating any user should not trigger any requests. | :green_circle: | :green_circle: | :green_circle: |

<summary>Delete a user that is above a reserved user in the table, and see that the tooltip does not appear.</summary>
<details>
Captura de pantalla 2023-07-06 a la(s) 16 32 51
Captura de pantalla 2023-07-07 a la(s) 08 20 19
</details>

<summary>Opening the flyout to edit a user and closing it without making any changes should not trigger any requests.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/de0f25c8-e9ac-4853-9e43-7795dc96dc95


</details>

<summary>Opening the flyout to create a user and closing it without creating any user should not trigger any requests. </summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/7e9400b1-c4d9-4f1c-a3d4-8c153d952095


</details>

<summary>Delete a role that is above a reserved user in the table, and see that the tooltip does not appear.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/e98b1faf-7b58-42fa-8901-64353a3dfd37


</details>

<summary>Opening the flyout to edit a role and closing it without making any changes should not trigger any requests.</summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/73fff095-7be7-4774-9180-d08ab2f2b660




</details>

<summary>Opening the flyout to role a user and closing it without creating any user should not trigger any requests. </summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/901037c9-eb06-44c2-acf3-9cc797dca679




</details>

<summary>Delete a policy that is above a reserved user in the table, and see that the tooltip does not appear.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/ae8ccfd8-308b-42ad-9487-5e32d1c094e9


</details>

<summary>Opening the flyout to edit a policy and closing it without making any changes should not trigger any requests.</summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/37eb29f5-5ae6-42f3-9ebb-7a6583d2cd56




</details>

<summary>Opening the flyout to create a policy and closing it without creating any user should not trigger any requests.</summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/31c8142b-2528-4b58-8298-97a2390146bc




</details>

<summary>Delete a roles mapping that is above a reserved user in the table, and see that the tooltip does not appear.</summary>
<details>

https://github.com/wazuh/wazuh-kibana-app/assets/99441266/482ced6d-36a7-43e7-93fd-61502deafae1


</details>

<summary>Opening the flyout to edit a roles mapping and closing it without making any changes should not trigger any requests.</summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/3f7430ac-36d3-4907-a084-c248b8665466




</details>

<summary>Opening the flyout to create a roles mapping and closing it without creating any user should not trigger any requests.</summary>
<details>


https://github.com/wazuh/wazuh-kibana-app/assets/99441266/a60e0b79-29a1-4ef5-8654-382ef63b5fc6




</details>

**Pull request merged**